services/galley/test/integration/API/MLS.hs

```diff
@@ -2210,6 +2211,42 @@ testGetGlobalTeamConv setup = do
   let cm = Aeson.decode rs :: Maybe GlobalTeamConversation
   liftIO $ assertEqual "conversation metadata" cm (Just expected)
 
+testConvListINotncludesGlobalV1 :: TestM ()
+testConvListINotncludesGlobalV1 = do
+  aliceQ <- randomQualifiedUser
+  let alice = qUnqualified aliceQ
+  tid <- createBindingTeamInternal "sample-team" alice
+  team <- getTeam alice tid
+  assertQueue "create team" tActivate
+  liftIO $ assertEqual "alice" alice (team ^. teamCreator)
+  assertQueueEmpty
+
+  -- global team conv doesn't yet include user
+  let paginationOpts = GetPaginatedConversationIds Nothing (toRange (Proxy @5))
+  listConvIds alice paginationOpts !!! do
+    const 200 === statusCode
+    const (Just [globalTeamConv tid]) =/~= (rightToMaybe . (<$$>) qUnqualified . decodeQualifiedConvIdList)
+
+  -- add user to conv
+  runMLSTest $ do
+    alice1 <- createMLSClient aliceQ
+
+    let response = getGlobalTeamConv alice tid <!! const 200 === statusCode
+    Just rs <- responseBody <$> response
+    let (Just gtc) = Aeson.decode rs :: Maybe GlobalTeamConversation
+        gid = cnvmlsGroupId $ gtcMlsMetadata gtc
+
+    void $ uploadNewKeyPackage alice1
+
+    -- create mls group
+    createGroup alice1 gid
+    void $ createAddCommit alice1 [] >>= sendAndConsumeCommitBundle
+
+  -- Now we should have the user as part of that conversation also in the backend
```

Comment:
this comment should read differently, I think

Reply:
Other way around, it should list for this user, since we're calling the right version of the API, no?